R/g.part1.R

```diff
@@ -392,8 +391,39 @@ g.part1 = function(datadir = c(), outputdir = c(), f0 = 1, f1 = c(),
   
   #--------------------------------------------------------------------------------
   # Run the code either parallel or in serial (file index starting with f0 and ending with f1)
- 
```

Comment:
Previously, all of this work was wrapped in an if statement so it only ran if parallel processing was used. With this revision, all of this work (including showing a warning if the number of metrics is high) is done regardless. Is that intended?

Reply:
ok, yes I see, I will correct it now

Reply:
Thanks @tarensanders for you help in recent weeks... having an extra critical pair of eyes really helps. By the way I already added you as package contributor to the DESCRIPTION file.

I made the 2.7-0 release this week because GGIR was listed for archival by CRAN as a result of issues with one of its dependencies. This was a bit in a rush by which I had less time to carefully test everything. My plan is to submit a new GGIR release to CRAN in the upcoming days to address the OSx issue + this PR.

Reply:
> Thanks @tarensanders for you help in recent weeks

More than happy to help, Vincent.

> By the way I already added you as package contributor to the DESCRIPTION file.

Thank you, I appreciate that.